ui/tests/helpers/clients.js

```diff
@@ -66,7 +66,6 @@ export const SELECTORS = {
       plotPoint: '[data-test-line-chart="plot-point"]',
     },
   },
-  emptyStateTitle: '[data-test-empty-state-title]',
```

Comment:
this was redundant since the same property is added on line 27 above inside the `...GENERAL`

Reply:
thank you for this cleanup! I've been pushing for the migration to use `general-selectors` 🧹